src/main/java/mate/academy/spring/dto/request/MovieRequestDto.java

```diff
@@ -1,7 +1,11 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+
 public class MovieRequestDto {
+    @NotNull
     private String title;
+    @NotNull
```

Comment:
```suggestion
    @NotBlank
```
that was comment from my work